### PR TITLE
Charts: update 'Discrete Aggregation' demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Charts/DiscreteAggregation/description.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
-The aggregation algorithm depends on the **aggregateByCategory** property. In this demo, this property is set to **true** and applies the *"sum"* [aggregation method](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#method) for the groups specified in the **dataSource**. 
+To [aggregate](/Documentation/Guide/UI_Components/Chart/Data_Aggregation/) points on discrete axes, do the following steps:
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/DiscreteAggregation/description.md]:
```suggestion
To [aggregate](/Documentation/Guide/UI_Components/Chart/Data_Aggregation/) points on discrete axes, follow the steps below:
```

File: JSDemos/Demos/Charts/DiscreteAggregation/description.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+1. Enable aggregation.    
+You can enable data aggregation for individual series, all series of a specific type, or for all series in the Chart. Assign `true` to the [aggregation](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/).[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#enabled) property.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/DiscreteAggregation/description.md]: Once I set the `aggregation.enabled` property to `true`, what does it do from the things you listed in the previous sentence? Does this action enable data aggregation for all series in the Chart? It's unclear.

File: JSDemos/Demos/Charts/DiscreteAggregation/description.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+In this demo, data aggregation is enabled for all series.
+
+2. Specify aggregation [method](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#method) (optional).
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/DiscreteAggregation/description.md]: Specify "some", "certain", or "any" aggregation method. In this particular case, we deal with indefiniteness.

```suggestion
2. Specify an aggregation [method](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#method) (optional).    
```

**Review by dmitry-eliseev-devexpress — COMMENTED** [review]
This is a good topic, however, I didn't understand what does the `aggregation`.`enabled` property does in the described context. It's obvious that it enables aggregation. I would have had no questions without the preceding sentence but I have them now. The first sentence in the "Enable aggregation" list item raises more questions than answers -- how should I do it all? How should I switch between these "aggregation modes"? What will I get exactly once I set the `aggregation.enabled` property to `true` as suggested?

I'll approve this topic as soon as I get an answer for my question. Thanks in advance.

File: JSDemos/Demos/Charts/DiscreteAggregation/description.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+To [aggregate](/Documentation/Guide/UI_Components/Chart/Data_Aggregation/) points on discrete axes, follow the steps below:
+
+1. Enable aggregation.    
+You can enable data aggregation for the following series:
+
+    - Individual series    
+    Assign `true` to the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[aggregation](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/aggregation/).[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/aggregation/#enabled) property.
+
+    - All series of a specific type  
+    Assign `true` to the `aggregaion.enabled` property belonging to the configuration object of a [specific series type](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/).
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/DiscreteAggregation/description.md]:
```suggestion
    Assign `true` to the `aggregaion.enabled` property that belongs to the configuration object of a [specific series type](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/).
```

File: JSDemos/Demos/Charts/DiscreteAggregation/description.md [hunk 5]
```diff
@@ -1,3 +1,21 @@
...
+
+    - All series in the Chart   
+    Assign `true` to the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[aggregation](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/).[enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/aggregation/#enabled) property.    
+
+The order of this list specifies the priority order of aggregation in the component. For example, you can enable aggregation for all series and disable it for a specific series.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/DiscreteAggregation/description.md]: > The order of **this list** specifies the priority order of aggregation in the component.

Of what list do we speak? You indirectly imply the three entries above, I understand. Example is good. Still, I would have preferred a more direct explanation. Something like that (you can come up with something even better than that):

```suggestion
Individual series aggregation settings have priority over common aggregation settings in the component. For example, you can enable aggregation for all series and disable it for a specific series.
```

Thing is that all indirect implications are not good. We should strive to make our texts as clear as possible. I use the following guiding principle in my writing: Every word that has potential for being misunderstood - will certainly be misunderstood. And I recommend you to do the same.

**Review by dmitry-eliseev-devexpress — DISMISSED** [review]
Yes, your change makes things much clearer. Thank you.

I approve the pull request, albeit I added a couple of minor changes. I hope they make sense.

File: JSDemos/Demos/Charts/DiscreteAggregation/description.md (hunk 2 above)
Comment (dmitry-eliseev-devexpress) [reply JSDemos/Demos/Charts/DiscreteAggregation/description.md]: I mark this thread as resolved. Thank you.

**Review by dmitry-eliseev-devexpress — APPROVED** [review]

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**